README.md

```diff
@@ -15,13 +15,13 @@ This project aims to provide:
 * An easy way to allow 3rd party developers to integrate into this stack to achieve interoperability between different [GObject] based libraries.
 
 Currently supported libraries
-* [GTK] (wip): UI-Toolkit (GTK3 + experimental GTK4)
-* [WebKitGTK] (planed): Browser-Engine
-* [JavaScriptCore] (planed): Javascript integration for [WebKitGTK]
-* [libhandy] (planed): Convergent UI-Widgets for [GTK] to support mobile phones
-* [dbus] (wip): Library for inter-process communication via [GIO]
-* [libchamplain] (planed): Library to display maps
-* [gstreamer] (wip): Multimedia Framework
+* [GTK] (partial support): UI-Toolkit (GTK3 + experimental GTK4)
+* [GStreamer] (partial support): Multimedia Framework
+* [DBus] (wip): Library for inter-process communication via [GIO]
```

Comment:
DBus should probably be marked with "partial support", too?

Reply:
I left it as-is because I'm not really sure how much/little we support it, other than the simple DBus sample. I'll go ahead and change it to partial though.